IPython/terminal/ptshell.py

```diff
@@ -232,7 +230,10 @@ def _(event):
         self._style = self._make_style_from_name(self.highlighting_style)
         style = DynamicStyle(lambda: self._style)
 
+        editing_mode = EditingMode.VI if self.vi_mode else EditingMode.EMACS
```

Comment:
OK. Since it's simple to do it now, I think I'll turn `vi_mode=True` into `edit_mode='vi'` in a separate PR.